index.md

```diff
@@ -12,23 +12,39 @@ Embedding Tableau content allows you to add the power of interactive visualizati
+
+***NOTE:** Connected Apps is available for both Tableau Server and Tableau Online using the [Embedding API v3](https://help.tableau.com/current/api/embedding_api/en-us/index.html).*
+
+**[External Authorization Servers (EAS)](https://help.tableau.com/current/server/en-us/connected_apps_eas.htm)** - Establishes a trust relationship between Tableau Server and the EAS. By establishing a trust relationship, you’re able to provide your users a single sign-on (SSO) experience to Tableau content embedded in your external applications through the identity provider (IdP) you’ve already configured for Tableau Server. When embedded Tableau content is loaded in your custom application, a standard OAuth flow is used. After users successfully sign in to the IdP, they are then automatically signed in to Tableau Server. To register an EAS with Tableau Server, you must have an EAS already configured. In addition, the EAS must send a valid [JSON web token (JWT)](https://datatracker.ietf.org/doc/html/rfc7519).
+
+***NOTE:** EAS is only available for Tableau Server and the [Embedding API v3](https://help.tableau.com/current/api/embedding_api/en-us/index.html).*
+
+**[Trusted Authentication](https://help.tableau.com/current/server/en-us/trusted_auth.htm)** - Establishes a trusted relationship between Tableau Server and an external application to provide a secure and seamless authentication experience. Unlike Connected Apps or External Authorization Server, Trusted Authentication does not follow the [JWT standard](https://datatracker.ietf.org/doc/html/rfc7519).
+
+If you are starting a new project, we recommend using Connected Apps or External Authorization Server.
+
+***NOTE:** Trusted Authentication is only available for Tableau Server and the [JavaScript API v2](https://help.tableau.com/current/api/js_api/en-us/JavaScriptAPI/js_api.htm).*
+
+**[SAML, OpenID, Active Directory, Kerberos](https://help.tableau.com/current/server/en-us/security_auth.htm)** - For environments that already use one of these systems, they can be leveraged to achieve Single Sign-On (and to leverage database security in the case of Kerberos).
+
+**Tableau Viz Lightning web component** - An easy and flexible way to quickly bring Tableau visualizations into Salesforce. Drag and drop the Tableau Viz Lightning web component in any Salesforce page, and paste the embedded URL of any Tableau view. Uses the Tableau permissions you have set up. [Embedding Tableau views into Salesforce](https://help.tableau.com/current/pro/desktop/en-us/embed_ex_lwc.htm) gives you the insight of Tableau dashboards where sales people are spending their time, filtered to the correct account/opportunity/etc.
```

Comment:
@d45 can you remove
"Uses the Tableau permissions you have set up."
and
"where sales people are spending their time" as salesforce is not only for sales people. thanks

Reply:
That section was redone and we dropped those lines.

**[Tableau Viz Lightning web component](https://www.tableau.com/products/viz-lightning-web-component-salesforce)** - Makes it easy to [Embed Tableau views into Salesforce Lightning pages](https://help.tableau.com/current/pro/desktop/en-us/embed_ex_lwc.htm). The Tableau Viz Lightning web component is available from the Salesforce AppExchange. To embed a view, you simply drag and drop the Tableau Viz Lightning web component onto the page and then provide the URL for the Tableau view. You can embed Tableau views from Tableau Server, Tableau Online, or Tableau Public.